src/services/template_converter_service.py

```
@@ -0,0 +1,105 @@
+            result = subprocess.run(
+                command,
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+
+            if temp_pdf_path.exists():
+                pdf_bytes = temp_pdf_path.read_bytes()
+                return pdf_bytes
+            else:
+                raise Exception("PDF generation failed")
```

Ruff (0.12.2)

Create your own exception

(TRY002)